lib/improver/utilities/cube_manipulation.py

```diff
@@ -320,22 +338,18 @@ class MergeCubes():
     """
     Class adding functionality to iris.merge_cubes()
 
-    Accounts for differences in attributes and coordinates to avoid merge
-    failures and anonymous dimensions.
+    Accounts for differences in attributes, cell methods and bounds ranges to
+    avoid merge failures and anonymous dimensions.
     """
     def __init__(self):
         """Initialise constants"""
         # List of attributes to remove silently if unmatched
         self.silent_attributes = ["history", "title", "mosg__grid_version"]
-        # List of coordinates that must strictly match on the input cubes.  If
-        # unmatched, an exception will be raised.
-        self.coord_mismatch_error_keys = ["threshold"]
 
     def _equalise_cubes(self, cubelist):
         """
-        Function to equalise cubes where the attributes, coordinates or cell
-        methods do not match.  Note this function cannot equalise cubes where
-        different coordinates are present on each cube.
+        Function to equalise cubes where the attributes or cell methods do not
+        match.
```

Comment:
This function actually serves very little purpose now, it simply wraps calls to 3 other functions, and these could just as easily be put into process in sequence; in fact that would make it easier to navigate this plugin without quite so much scrolling up and down. As it's a private function we are not intending to use it elsewhere, so I would recommend getting rid of it.

Reply:
:+1:

Reply:
Hooray for more deletions. Thanks for doing this.